frontend/src/pages/orders/OrderList.tsx

```diff
@@ -0,0 +1,118 @@
+            setOrders(response.data);
+            setOrdersByStatus(response.data);
+        });
+    }, []);
+
+    return (
+        <div className={'container'}>
+            <PageHeader title="Bestellliste" pageType="order"/>
+
+            <div className="form-check form-check-inline mb-3">
+                <p>Bestellstatus:</p>
+            </div>
+
+            <div className="form-check form-check-inline mb-3">
+                <input className="form-check-input" type="radio" name="inlineRadioOptions" id="allOrders"
+                       value="allSuppliers" onClick={handleStatusChange} defaultChecked/>
+                <label className="form-check-label" htmlFor="allSuppliers">Alle Bestellungen</label>
+            </div>
+
+            <div className="form-check form-check-inline mb-3">
+                <input className="form-check-input" type="radio" name="inlineRadioOptions" id="PENDING"
+                       value="PENDING" onClick={handleStatusChange}/>
+                <label className="form-check-label" htmlFor="activeSuppliers">Ausstehend</label>
+            </div>
+            <div className="form-check form-check-inline mb-3">
+                <input className="form-check-input" type="radio" name="inlineRadioOptions" id="CONFIRMED"
+                       value="CONFIRMED" onClick={handleStatusChange}/>
+                <label className="form-check-label" htmlFor="passiveSuppliers">Bestätigt</label>
+            </div>
+            <div className="form-check form-check-inline mb-3">
+                <input className="form-check-input" type="radio" name="inlineRadioOptions" id="SHIPPED"
+                       value="SHIPPED" onClick={handleStatusChange}/>
+                <label className="form-check-label" htmlFor="passiveSuppliers">Ausgeliefert</label>
+            </div>
+            <div className="form-check form-check-inline mb-3">
+                <input className="form-check-input" type="radio" name="inlineRadioOptions" id="DELIVERED"
+                       value="DELIVERED" onClick={handleStatusChange}/>
+                <label className="form-check-label" htmlFor="passiveSuppliers">Geliefert</label>
+            </div>
+            <div className="form-check form-check-inline mb-3">
+                <input className="form-check-input" type="radio" name="inlineRadioOptions" id="CANCELLED"
+                       value="CANCELLED" onClick={handleStatusChange}/>
+                <label className="form-check-label" htmlFor="passiveSuppliers">Abgesagt</label>
+            </div>
+
+            <div className="input-group">
+                <span className="input-group-text" id="basic-addon3">Schreiben einen Kundennamenfilter</span>
+                <input
+                    type="text"
+                    className="form-control"
+                    id="basic-url"
+                    aria-describedby="basic-addon3 basic-addon4"
+                    onChange={(e) => setFilter(e.target.value)}
```

Comment:
a better practice would be, to write a seperate "setFilter" function and just call the function here. You can also do it like this, since it's a very small line, but I would still suggest you, to keep your logic above the "return". This way, you can better see, where for example you are actually setting your "filter" useState variable with "setFilter".

Reply:
I called setSelectedCustomerName in the handleStatusChange function and created a separate function called handleSelectedCustomerName.